Game/AI/Decks/YubelExecutor.cs

```diff
@@ -0,0 +1,2315 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Claims;
+using WindBot;
+using WindBot.Game;
+using WindBot.Game.AI;
+using YGOSharp.Network.Enums;
+using YGOSharp.OCGWrapper;
+using YGOSharp.OCGWrapper.Enums;
+
+namespace WindBot.Game.AI.Decks
+{
+    [Deck("Yubel", "AI_Yubel", "Fiendsmith Yubel")]
```

Comment:
The 3rd param should be "level"

https://github.com/IceYGO/windbot/blob/7d0a5a85a848b0b3fce45416d2e2174d89ffb9e6/Game/AI/DeckAttribute.cs?plain=1#L12

currently only used here, for random
https://github.com/IceYGO/windbot/blob/7d0a5a85a848b0b3fce45416d2e2174d89ffb9e6/Game/AI/DecksManager.cs?plain=1#L69